SurveyApp.DAL/Repositories/SurveyRepository.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using SurveyApp.DAL.DataContext;
+using SurveyApp.DAL.EntityModels;
+using SurveyApp.DAL.Interfaces;
+
+namespace SurveyApp.DAL.Repositories
+{
+    public class SurveyRepository : Repository<SurveyDataModel>, ISurveyRepository
+    {
+
+        public ApplicationContext Database { get; set; }
```

Comment:
It seems that public ApplicationContext Database { get; set; } is duplicated. Base repository contains the same property.
Can it be deleted?

Reply:
Redundant creating of ApplicationContext was deleted. Moreover, the Singleton was implemented in ApplicationContext for resources saving.